cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowContext.java

```diff
@@ -64,11 +65,11 @@
   Map<String, WorkflowNodeState> getNodeStates();
 
   /**
-   * Return true if the execution was successful, false otherwise. This method can be
-   * used from {@link AbstractWorkflow#destroy} to determine the status of the {@link Workflow}.
-   * It can also be used from {@link WorkflowAction#destroy} method to determine the status
-   * of the {@link WorkflowAction}. If it is called before the execution is completed, false is returned.
+   * Return the state of the workflow. This method can be used from {@link AbstractWorkflow#destroy}
+   * to determine the status of the {@link Workflow}. It can also be used from {@link WorkflowAction#destroy} method
+   * to determine the status of the {@link WorkflowAction}.
```

Comment:
better to add description about `CustomAction` since `WorkflowAction` is deprecated.